src/Components/Bookpage/BookDetails/index.js

```diff
@@ -1,3 +1,12 @@
+
+import { useState } from "react"
+import Reviews from "./Reviews"
+import "./bookdetails.css"
+
+function BookDetails({ image, title, author, year, pageCount, genre, blurb, reviews }) {
+    const totalScore = reviews.reduce((ratingSum, review) => ratingSum + review.rating, 0)
+    const avgScore = totalScore / reviews.length
+
 import ClaimForm from './ClaimForm'
 import ReturnForm from './ReturnForm'
 import './bookdetails.css'
```

Comment:
This section is breaking the whole app. imports need to be at the top - looks like a merge conflict that's been messed up slightly

Reply:
fixed